addons/payment/models/payment_token.py

```diff
@@ -20,7 +20,7 @@ class PaymentToken(models.Model):
     acquirer_id = fields.Many2one(
         string="Acquirer Account", comodel_name='payment.acquirer', required=True)
     company_id = fields.Many2one(  # Indexed to fasten ORM searches on c_id (from ir_rule or other)
-        related='acquirer_id.company_id', store=True, index=True)
+        related='acquirer_id.company_id', index=True)
```

Comment:
index is useless on unstored field ;)

Reply:
https://github.com/odoo/odoo/pull/56187#discussion_r529675382 :thinking: 

I assumed from your comment that `index=True` **implied** `store=True`